device-data/types/basal/suppressed.md

```diff
@@ -1,5 +1,279 @@
 <!-- non-generated document! all areas editable -->
 
-Coming soon!
+## the `suppressed` field on `temp`s and `suspend`s
 
-<!-- TODO: document how `suppressed` works when a `temp` or `suspend` basal crosses > 1 segment of a basal schedule -->
\ No newline at end of file
+Some insulin pump data protocols provide enough information for us to track various aspects (e.g., `rate`, `scheduleName`) of the basal that *would have been in effect* had the currently active `temp` or `suspend` basal **not** been programmed (or triggered). Where this information is available, we provide it in as much detail as possible as an embedded object in the `suppressed` field on a `temp` or `suspend` basal interval.
+
+In general, this `suppressed` object need and **must** only contain the bare minimum of information:
+- the `type` (= `basal`)
+- the `deliveryType` (only `scheduled` if the currently active basal is a `temp`; otherwise (i.e., if the currently active basal is a `suspend`), may be `scheduled` *or* `temp`)
```

Comment:
I suspect that the feedback here is that the proposal needs to do a better job of explaining that the restrictions described are **not** true of jellyfish-ingested data (and thus much production data) because `suppressed` basically wasn't validated at all (beyond it had to be an object). Is your objection really that the data model should be different or that I need to do a waaaay better job of calling out that this doc mostly describes platform-only (proposed) constraints, in my interpretation of what the actual intent of the data model is?

Also, production data includes known *very* bad data (like that uploaded during **early-stage development** of the Tandem driver by contractors who weren't super familiar with our data model), so in general I'm not inclined to care much about any old thing that exists in production data. Do you have a different instinct there?

Your example here is CareLink, of course, but that's also code that's on its way out...